.clang-format

```diff
@@ -111,7 +109,7 @@ AlignOperands:  AlignAfterOperator
 #   #endif
 #   #endif
 #
-IndentPPDirectives: AfterHash
+IndentPPDirectives: None
```

Comment:
The majority of the preprocessor directives are just header and footer guards - so this makes sense to drop this formatting.

In a couple PPD-heavy areas, this "meta-formatting" helped reveal the pre-processor logic, like in `core_dyn_x86`:

```text
src/cpu/core_dyn_x86.cpp:#include "dyn_cache.h"
src/cpu/core_dyn_x86.cpp:#      if defined(X86_DYNFPU_DH_ENABLED)
src/cpu/core_dyn_x86.cpp:#      endif
src/cpu/core_dyn_x86.cpp:#      define X86    0x01
src/cpu/core_dyn_x86.cpp:#      define X86_64 0x02
src/cpu/core_dyn_x86.cpp:#      if C_TARGETCPU == X86_64
src/cpu/core_dyn_x86.cpp:#              include "core_dyn_x86/risc_x64.h"
src/cpu/core_dyn_x86.cpp:#      elif C_TARGETCPU == X86
src/cpu/core_dyn_x86.cpp:#              include "core_dyn_x86/risc_x86.h"
src/cpu/core_dyn_x86.cpp:#      else
src/cpu/core_dyn_x86.cpp:#              error DYN_X86 core not supported for this CPU target.
src/cpu/core_dyn_x86.cpp:#      endif
src/cpu/core_dyn_x86.cpp:#include "core_dyn_x86/decoder.h"
src/cpu/core_dyn_x86.cpp:#      if defined(X86_DYNFPU_DH_ENABLED)
src/cpu/core_dyn_x86.cpp:#      else
src/cpu/core_dyn_x86.cpp:#      endif
src/cpu/core_dyn_x86.cpp:#      if defined(X86_DYNFPU_DH_ENABLED)
src/cpu/core_dyn_x86.cpp:#      endif
```

(just a comment, no need for changes)

Reply:
No silver bullet unfortunately :(